doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-dmn-bpmn-maven-create.adoc

```diff
@@ -0,0 +1,51 @@
+[id="proc-dmn-bpmn-maven-create_{context}"]
+= Creating and executing DMN and BPMN models using Maven
+
+You can use Maven archetypes to develop DMN and BPMN models and integrate the archetypes with your {PRODUCT} decision and process services. This method of developing DMN and BPMN models is helpful for building new business applications using the {PRODUCT} VSCode extension.
```

Comment:
@hmanwani-rh So as we start introducing alternative or supplemental development features from Kogito like this, we just need to make it clear how things fit in to the story, especially relative to Business Central. So for this intro paragraph, I suggest something like this:

> You can use Maven archetypes to develop DMN and BPMN models in VSCode using the {PRODUCT} VSCode extension instead of in Business Central. You can then integrate your archetypes with your {PRODUCT} decision and process services in Business Central as needed.

OR

>If you plan to develop DMN and BPMN models in VSCode using the {PRODUCT} VSCode extension instead of in Business Central, you must create a Maven archetype where you can design your models. You can then integrate your archetypes with your {PRODUCT} decision and process services in Business Central as needed.

Or something like that.

Also, for the chapters that follow this in both the GSGs, I suggest clarifying the titles as follows. This is also consistent to how we clarify it in the developing docs as well:

* Creating the traffic violations project **in Business Central**
* Creating the mortgage-process project **in Business Central**

Lastly, I don't see a reason not to include this proc in the developing assemblies as well. I see you've just included the cross-ref to the GSGs which there contain the section, but I think it's just as pertinent directly in the two dev assemblies too, as includes. If anything, I think it's actually even better to do this in reverse: Include it in the dev assemblies then in the GSGs have a cross-ref to the sections in the dev assemblies. Reason is that the GSGs are intentionally a single-track tutorial that chooses a method/scenario and runs with it from beginning to end. While the dev assemblies are there to give you the various options. So the maven approach really isn't even part of the GSG tutorial scenario, so I don't think it actually fits the story per se. Just my thoughts, whatever you think.

Reply:
I would go with the first suggestion. 
Also, I've included the maven project section in all four assemblies: Getting started guides (decision and process services) and developing guides (decision and process services).